### ISSUE TITLE
A question about resolution

### ISSUE THREAD

**Issue by alizezhu** [MIC-DKFZ/nnUNet#228]
Hi Fabian,
For the training data to be put in the rawdata directory, shall I manually normalize their spatial resolution? Or it will be automatically taken care as the information is already in the header of nifti?
Thanks a lot for your great work!

**Comment by FabianIsensee** [comment]
This is not just taken care of, this is an essential part of nnU-Net. Never resize your images!
See the Methods section in the paper :-)
